pkg/tfgen/docs_test.go

```diff
@@ -624,6 +624,38 @@ func TestArgumentRegex(t *testing.T) {
 	}
 }
 
+func TestArgumentRegexAuto(t *testing.T) {
+	tests := []struct {
+		name     string
+		input    []string
+		expected autogold.Value
+	}{
+		{
+			name: "scaleway bucket_panic",
```

Comment:
Instead of calling it `scaleway bucket_panic`, can we name the test case something descriptive?

Likewise, I have no idea what `TestArgumentRegexAuto` is talking about, since we don't change the any regexp to put newly under test.

Reply:
Renamed the test.

TestArgumentRegexAuto is a variant of TestArgumentRegex, which uses autogold

Reply:
Feel free to suggest new names for either, I've followed the existing naming the the file.